Frontend/src/Components/GeneralData/IconCell.js

```diff
@@ -0,0 +1,7 @@
+import { Box, Flex, Image} from "@chakra-ui/react";
+
+export default function IconCell(props) {
+    return(
+        <Flex>{props.boolean ? <Image boxSize={props.boxSize} src={props.image}/> : <Box h={props.boxSize}></Box>}</Flex>
```

Comment:
```suggestion
        <>{props.boolean ? <Image boxSize={props.boxSize} src={props.image}/> : <Box h={props.boxSize} />}</>
```

Reply:
Since this is only one line and the only common/generic attribute is boxSize, it would be better to just use this line directly in Driver_Comms.js (in which case the empty outer tags would be unnecessary) rather than having this custom component.